Feature/issue57/add assert alert body (#145)
theappbusiness/TABTestKit: CHANGELOG.md

    - Added `carthage.sh` to fix the issue with duplicated architecture when Carthage needs to build with Xcode 12.0
- Fixed Carthage issue to allow `Biometics` framework to be seperated from TABTestKit as the same of SwiftPM
- Fixed a timezone issue on TestExample when switching from Summer Time to Winter Time. We make sure that the DatePicker are now always in UTC Timezone

> - Added fun message to assert alert message body

Reviewer: ```suggestion
- Added a method to assert the alert message
```
